ec2/spark_ec2.py

```diff
@@ -330,7 +338,10 @@ def get_spark_ami(opts):
         print >> stderr,\
             "Don't recognize %s, assuming type is pvm" % opts.instance_type
 
-    ami_path = "%s/%s/%s" % (AMI_PREFIX, opts.region, instance_type)
+    # URL prefix from which to fetch AMI information
+    ami_prefix = "{r}/{b}/ami-list".format(r=opts.spark_ec2_git_repo.replace("github","raw.github"), b=opts.spark_ec2_branch)
```

Comment:
Good point. 
replace doesn't seem to support regexes, so to keep things simple I matched like you suggested, and limited to 1 match